### PR TITLE
maybe there are two description mistake in the plugin.md file

### PR DESCRIPTION
**Pull request by Senasiko**
I found two description mistake when I read the plugin.md
1.  `export funtion pluginWillUnload()` -> `export function pluginWillUnload()`
2.  the `typeof footer` in `window.toggleModal` should be `array`

### CODE REVIEW

File: docs/plugin.md [hunk 1]
```diff
@@ -760,4 +762,4 @@ More plugins are available as reference on poooi(https://github.com/poooi)
 
 
 ## Community
-Welcome to become developer for poi. You may post your questions and ideas as github issues, or join our [telegram group](https://telegram.me/joinchat/AoMUpkCr6B8uH7EUewq6eQ) for discussions.
\ No newline at end of file
+Welcome to become developer for poi. You may post your questions and ideas as github issues, or join our [telegram group](https://telegram.me/joinchat/AoMUpkCr6B8uH7EUewq6eQ) for discussions.
```
Comment (KagamiChan) [inline docs/plugin.md]: What is the difference? line feed?

File: docs/plugin-cn.md [hunk 2]
```diff
@@ -252,11 +252,13 @@ window.notify('Something'); // 桌面通知
...
-  func: Function, // 点击按钮后的动作
-  style: String in ['default', 'primary', 'success', 'info', 'danger', 'warning'] // 颜色
-}
+var footer = [
+  {
+    name: String, // 按钮显示的名字
+    func: Function, // 点击按钮后的动作
+    style: String in ['default', 'primary', 'success', 'info', 'danger', 'warning'] // 颜色
+  }  
+];
```
Comment (KagamiChan) [inline docs/plugin-cn.md]: tailing semicolon is forbidden

**Comment by KagamiChan** [comment]
It is true that the dev doc is not of best quality. Any improvement is appreciated

File: docs/plugin.md (hunk 1 above)
Comment (Senasiko) [reply docs/plugin.md]: sorry, I don't know, maybe because of the text editor

**Comment by KagamiChan** [comment]
Since we did not set a gitattribute for .md files, this is OK. Please fix the semicolon so that this could get merged

**Comment by Senasiko** [comment]
@KagamiChan All right. I have fixed the semicolon. Thanks.

**Pull request merged**